src/test/java/seedu/mindmymoney/command/ListCommandTest.java

```diff
@@ -0,0 +1,28 @@
+package seedu.mindmymoney.command;
+
+import org.junit.jupiter.api.Test;
+import seedu.mindmymoney.MindMyMoneyException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ListCommandTest {
+    @Test
+    void listToString_happyPath_expectString() {
```

Comment:
```suggestion
    void listToString_multipleInputs_expectString() {
```
Maybe change happyPath to something more descriptive?